x-pack/docs/en/security/authorization/mapping-roles.asciidoc

```diff
@@ -10,6 +10,9 @@ For other types of realms, you must create _role-mappings_ that define which
 roles should be assigned to each user based on their username, groups, or
 other metadata.
 
+NOTE: When <<anonymous-access,anonymous access>> is enabled, the roles
```

Comment:
These are two unrelated statements that both fit a note section. I think that merging them make doesn't make much sense.

Reply:
Thanks @jkakavas -- I think you're right. I split the notes out into separate notes and moved the one about users with no roles after the next paragraph.